### ISSUE TITLE
issues about finalizing with run-asm-pipeline-post-review.sh

### ISSUE THREAD

**Issue by zjuzexian** [theaidenlab/3d-dna#149]
Hi all,
I meet a problem when finalizing the output after re-order the chromosome by juicer.
And every time when after filling gaps. (finalize-output.sh)
The program died with the temp.fasta collaped with error like this:
![f6a444044d147d03de19b88637910f1](https://user-images.githubusercontent.com/84708729/162428804-d86b615b-f866-4c10-95a2-3b6207653241.png)
![a273ce0332c7f297d920019a656be41](https://user-images.githubusercontent.com/84708729/162429618-6fc0ed9b-693e-4bba-8c28-60fc2f372653.png)
And the .FINAL.fasta is much smaller(900Mb) than the input .final.fasta(1.5G).

And I checked the  .final.cprops, it may died after the last scaf in temp.fa(scaf_30_RagTag:::fragment_5:::debris) and then reach these scaffolds:
scaf_30_RagTag:::fragment_6:::overhang_1 730 923994
scaf_30_RagTag:::fragment_6:::overhang_2:::gap 731 6
scaf_30_RagTag:::fragment_7:::debris:::overhang_1:::gap 732 94
scaf_30_RagTag:::fragment_7:::debris:::overhang_2 733 10906
And I checked in the .finale cprops there is no other scaffolds with "overhang", only fragment or debris.

I have checked that for .final.fasta, these sequneces(scaf_30_RagTag:::fragment_6:::overhang_1 toscaf_30_RagTag:::fragment_7:::debris:::overhang_2 ) are all exist.

I wonder why it happen and how can I fix this?

Thanks!

**Comment by zjuzexian** [comment]
It seems that the FINAL.assembly file before my adjust habe already had these fragment overhang, which doesn't exist in the previous .final.assembly file

**Comment by dudcha** [comment]
My guess is that you are reviewing the wrong file in Juicebox. JB should load rawchrom.hic and rawchrom.assembly (final.hic and final.assembly in previous versions). These files should not contain gapncontigs and overhangs, that’s one step before.

**Comment by CuiYutong916** [comment]
I met the same problem. Did you get the solution?
Thanks,

**Comment by zjuzexian** [comment]
> 

Yes, just as dudcha said, you should check if you are using the right file from juicebox.
